File: Java/T6/T6_InicioFG/app/src/main/java/com/example/t6_iniciofg/fragements/FragmentDos.java
package com.example.t6_iniciofg.fragements;

import android.content.Context;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.example.t6_iniciofg.R;

public class FragmentDos extends Fragment  {

    private String dato;
    private View view;
    private TextView texto;

    public static FragmentDos newInstance(String data){

        FragmentDos fragmentDos = new FragmentDos();
        Bundle bundle = new Bundle();
        bundle.putString("dato",data);
        fragmentDos.setArguments(bundle);
        return fragmentDos;

    }

    @Override
    public void onAttach(@NonNull Context context) {
        super.onAttach(context);
        if (this.getArguments()!=null){
            dato = this.getArguments().getString("dato");
        }
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        view = inflater.inflate(R.layout.fragment_dos_layout,container,false);
        return view;
    }

    @Override
    public void onStart() {
        super.onStart();
        texto = view.findViewById(R.id.etiqueta_fragment2);
        initGUI();
    }

    private void initGUI() {
        texto.setText(dato);
    }
}

File: Kotlin/T6/codigos_ces/iniciofg/app/src/main/java/com/example/iniciofg/ui/fragment/FragmentDos.kt
package com.example.iniciofg.ui.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.example.iniciofg.databinding.FragmentDosBinding
import com.example.iniciofg.databinding.FragmentUnoBinding

class FragmentDos: Fragment() {

    private lateinit var binding: FragmentDosBinding

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        binding = FragmentDosBinding.inflate(inflater,container,
            false)
        return binding.root
    }

}